endpoints/openrtb2/auction_test.go

```diff
@@ -6108,3 +6108,72 @@ func TestValidateAliases(t *testing.T) {
 func fakeNormalizeBidderName(name string) (openrtb_ext.BidderName, bool) {
 	return openrtb_ext.BidderName(strings.ToLower(name)), true
 }
+
+func Test_secCookieDeprecation(t *testing.T) {
```

Comment:
Nitpick: Test name should be `TestSetCookieDeprecation`.

Reply:
done